Generation/doc/AddingPileupToGeneration.md

```diff
@@ -0,0 +1,33 @@
+
+
+# Adding Pileup in FCCSW in Generation
```

Comment:
So, something like `PileupInFCCSW.md`, and then link to this file? This should definitely be added, but can the Generation part stay like this?